mdsuite/database/data_manager.py

```diff
@@ -309,6 +323,11 @@ def dictionary_generator(ensemble_loop, correlation_time, data_range):
             -------
             None
             """
+            ensemble_loop = int(
+                np.clip(
+                    (glob_data[b"data_size"] - data_range) / correlation_time, 1, None
```

Comment:
why is this a byte-string?

Reply:
When I pass dict objects through tensoflow I get back byte strings, I give it normal ones and get back bytes, no idea why, but it is what happens. This is related to #453